packages/reef-knot/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "reef-knot",
```

Comment:
If it's not scoped to reef-know, will it be published as personal library? And who will be publishing it?

Reply:
It is expected.
The main package is `reef-knot`
Others are scoped in `@reef-knot/*`
There are some open-source projects using this naming pattern.
It will be published using my personal account, since now we don't have any decisions and solutions to publish it in another way, only discussions. The rights to the package will be transferred later.